vllm/core/scheduler.py

```diff
@@ -1295,17 +1305,39 @@ def _schedule_default(self) -> SchedulerOutputs:
 
         # Merge lists
         num_prefill_groups = len(prefills.seq_groups)
+        ignored_seq_groups_for_embeds: List[SequenceGroup] = []
```

Comment:
```suggestion
        ignored_seq_groups_for_embeds = list[SequenceGroup]()
```

No need to repeat the type hint since https://peps.python.org/pep-0585/